d2l-typography.scss

```diff
@@ -1,169 +1 @@
-@import 'node_modules/d2l-colors/d2l-colors.scss';
-
-@mixin d2l-typography() {
-
-	color: $d2l-color-ferrite;
-	display: block;
-	font-family: 'Lato', 'Lucida Sans Unicode', 'Lucida Grande', sans-serif;
-	letter-spacing: 0.01rem;
-	font-size: 0.95rem;
-	font-weight: 400;
-	line-height: 1.4rem;
-
-	p {
-		margin: 1rem 0;
-	}
-
-	&.d2l-dyslexic,
-	& .d2l-dyslexic {
-		font-family: 'Open Dyslexic', sans-serif;
-		font-weight: 400;
-	}
-
-	&:lang(ar),
-	& :lang(ar) {
-		font-family: 'Arabic Transparent', 'Arabic Typesetting', 'Geeza Pro', sans-serif;
-	}
-
-	&:lang(zh),
-	& :lang(zh) {
-		font-family: 'Microsoft YaHei', 'Hiragino Sans GB', sans-serif;
-	}
-
-	&:lang(ko),
-	& :lang(ko) {
-		font-family: 'Apple SD Gothic Neo', Dotum, sans-serif;
-	}
-
-	&:lang(ja),
-	& :lang(ja) {
-		font-family: 'Hiragino Kaku Gothic Pro', 'Meiyro', sans-serif;
-	}
-
-}
-
-@mixin d2l-typography-font-face() {
-	@font-face {
-		font-family: 'Lato';
-		font-style: normal;
-		font-weight: 400;
-		src: url(https://s.brightspace.com/lib/fonts/0.4.0/assets/Lato-400.woff2) format('woff2'), url(https://s.brightspace.com/lib/fonts/0.4.0/assets/Lato-400.woff) format('woff');
-	}
-	@font-face {
-		font-family: 'Lato';
-		font-style: normal;
-		font-weight: 700;
-		src: url(https://s.brightspace.com/lib/fonts/0.4.0/assets/Lato-700.woff2) format('woff2'), url(https://s.brightspace.com/lib/fonts/0.4.0/assets/Lato-700.woff) format('woff');
-	}
```

Comment:
Are we ready to kill the OD assets from `d2l-typography`? I guess anything that'll pick up this change will be building/deploying for 20.19.8... so should be OK.